packages/appstore/src/components/onboarding-new/__tests__/static-dashboard.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import StaticDashboard from '../static-dashboard';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+describe('<StaticDashboard />', () => {
+    const is_blurry = {
+        icon: true,
+        item: false,
+        text: false,
+        get: false,
+        topup: false,
+        trade: false,
+        cfd_item: false,
+        cfd_text: false,
+        options_item: false,
+        options_text: false,
+        cfd_description: false,
+        options_description: false,
+        platformlauncher: false,
+    };
+
+    const is_onboarding_animated = {
+        text: false,
+        trade: false,
+        topup: false,
+        button: false,
+        get: false,
+    };
+
+    test('should render derivez in page if !CFDs_restricted_countries (non-eu countries)', async () => {
```

Comment:
why async function when we are not awaiting for anything
```suggestion
    test('should render derivez in page if !CFDs_restricted_countries (non-eu countries)', () => {
```